test/unit/lib/stringUtils.js

```diff
@@ -39,4 +39,34 @@ describe('stringUtils', () => {
       expect(word).to.equal('LS1');
     });
   });
+
+  describe('joinWithCommas', () => {
+    it('should return no commas or and for a single word', () => {
```

Comment:
For consistency with the last two tests, can the and be surrounded by `'`?

Reply:
removed code and tests as no longer used.